nicegui/static/templates/local/materialdesignicons/iconfont/README.md

```diff
@@ -1,10 +1,8 @@
 The recommended way to use the Material Icons font is by linking to the web font hosted on Google Fonts:
 
 ```html
-<link
```

Comment:
Do we need this formatting fix? Better to leave as much untouched as possible...